hpe3parclient/client.py

```diff
@@ -4208,3 +4208,56 @@ def resyncPhysicalCopy(self, volume_name):
         info = {'action': self.RESYNC_PHYSICAL_COPY}
         response = self.http.put("/volumes/%s" % (volume_name), body=info)
         return response[1]
+
+    def admitRemoteCopyLinks(self, targetName, port_address):
+        """Adding remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type - string
+        :port_address - Target system port address
+        :type- string
+        """
+        cmd = ['admitrcopylink', targetName, port_address]
+        return self._run(cmd)
+
+    def dismissRemoteCopyLinks(self, targetName, port_address):
+        """Adding remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type - string
+        :port_address - Target system port address
+        :type- string
+        """
+        cmd = ['dismissrcopylink', targetName, port_address]
+        return self._run(cmd)
+
+    def admitRemoteCopyTarget(self, targetName, mode, remote_copy_group_name, volumes_in_rcg=False, source_target_volume_pairs_list=None):
+        """Adding target to remote copy group
+        :param targetName - The name of target system
+        :type - string
+        :mode - synchronization mode
+        :type - string
+        :remote_copy_group_name
+        :type - string
+        :volumes_in_rcg
+        :type - boolean
+        :source_target_volume_pairs_list: list of primary and remote copy volumes
+        :type - list
+        """
+        if not volumes_in_rcg:
+            cmd = ['admitrcopytarget', targetName, mode, remote_copy_group_name]
```

Comment:
cmd = ['admitrcopytarget', targetName, mode, remote_copy_group_name] 

This can be placed above condition, no need to duplicate the statement